app/controllers/api/v1/checkout_controller.rb

```diff
@@ -2,11 +2,15 @@ class Api::V1::CheckoutController < Api::ApiController
 include OrderHelper
 
   def checkout 
+    # CON: Not using Rails permitted params, e.g.:
+    #
+    # params.require(:cart).permit(:reference, lineItems: [:name, :price, :collection])
```

Comment:
i love this way, usually i use this when i will save something on database with active record i not think use it in calculate api :D thankyou

Reply:
Gotcha! Any specific reason why you didn't use it here?

Reply:
Not exactly, it's just that I like to test new things, I love discovering new ways of doing things, ruby, and other languages also have several ways of doing the same thing, sometimes they are more complex, that's why I asked about the sum iteration hehe, but sometimes they are just pieces of a big fun puzzle to put together :D